client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
+export class KeyboardController extends Controller<HTMLButtonElement> {
+  static values = { key: String };
+
+  handleKey(event: Event) {
+    event.preventDefault();
+    this.element.click();
+  }
+
+  initialize(): void {
+    this.handleKey = this.handleKey.bind(this);
+  }
+
+  keyValueChanged(key, previousKey) {
+    if (previousKey && previousKey !== key) {
+      Mousetrap.unbind(previousKey, this.handleKey);
```

Comment:
```suggestion
      Mousetrap.unbind(previousKey);
```

There is no second param here that's the function, my bad (I think I had this on the POC). Have another read of the docs though. See https://craig.is/killing/mice#api.unbind

Reply:
Yes you're right. There's only a second param if type of event (`keyup`,`keydown`) is specified.